libafl/src/events/llmp/mgr.rs

```diff
@@ -410,25 +410,24 @@ where
             + EvaluatorObservers<E::Observers>
             + Evaluator<E, Self>,
     {
-        if !self.hooks.pre_exec_all(state, client_id, &event)? {
-            return Ok(());
-        }
+        let evt_name = event.name_detailed();
+
         match event {
             Event::NewTestcase {
                 input,
                 client_config,
                 exit_kind,
-                corpus_size: _,
                 observers_buf,
-                time: _,
-                executions: _,
+                #[cfg(feature = "std")]
                 forward_id,
+                ..
             } => {
-                log::info!("Received new Testcase from {client_id:?} ({client_config:?}, forward {forward_id:?})");
+                #[cfg(feature = "std")]
+                debug!("[{}] Received new Testcase {evt_name} from {client_id:?} ({client_config:?}, forward {forward_id:?})", std::process::id());
 
                 if self.always_interesting {
                     let item = fuzzer.add_input(state, executor, self, input)?;
-                    log::info!("Added received Testcase as item #{item}");
+                    debug!("Added received Testcase as item #{item}");
```

Comment:
Why debug? This seems pretty ugly / please use `log`

Reply:
i tried to keep the existing as log and the new one as debug to avoid polluting the existing logs, otherwise it becomes *very* verbous. this one is my mistake, i forgot to revert it to info